numpy/core/src/_simd/_simd_inc_easyintrin.h

```diff
@@ -0,0 +1,229 @@
+        };                                                \
+        return simd_arg_to_obj(&a);                       \
+    }
+
+#define SIMD_IMPL_INTRIN_0N(NAME)                         \
+    static PyObject *simd__intrin_##NAME                  \
+    (PyObject* NPY_UNUSED(self), PyObject *args)          \
+    {                                                     \
+        if (simd__no_arguments(                           \
+            args, NPY_TOSTRING(NAME)                      \
+        )) return NULL;                                   \
+        npyv_##NAME();                                    \
+        Py_RETURN_NONE;                                   \
+    }
+
+#define SIMD_IMPL_INTRIN_1(NAME, RET, IN0)                \
+    static PyObject *simd__intrin_##NAME                  \
+    (PyObject* NPY_UNUSED(self), PyObject *args)          \
+    {                                                     \
+        simd_arg arg = {.dtype = simd_data_##IN0};        \
+        if (!PyArg_ParseTuple(                            \
+            args, "O&:"NPY_TOSTRING(NAME),                \
+            simd_arg_converter, &arg                      \
+        )) return NULL;                                   \
+        simd_data r = {.RET = npyv_##NAME(                \
+            arg.data.IN0                                  \
+        )};                                               \
+        simd_args_sequence_free(&arg, 1);                 \
+        arg.data = r;                                     \
+        arg.dtype = simd_data_##RET;                      \
+        return simd_arg_to_obj(&arg);                     \
+    }
+
+#define SIMD_IMPL_INTRIN_2(NAME, RET, IN0, IN1)           \
+    static PyObject *simd__intrin_##NAME                  \
+    (PyObject* NPY_UNUSED(self), PyObject *args)          \
+    {                                                     \
+        simd_arg req_args[] = {                           \
+            {.dtype = simd_data_##IN0},                   \
+            {.dtype = simd_data_##IN1},                   \
+        };                                                \
+        if (!PyArg_ParseTuple(                            \
+            args, "O&O&:"NPY_TOSTRING(NAME),              \
+            simd_arg_converter, &req_args[0],             \
+            simd_arg_converter, &req_args[1]              \
+        )) return NULL;                                   \
+        simd_data r = {.RET = npyv_##NAME(                \
+            req_args[0].data.IN0,                         \
+            req_args[1].data.IN1                          \
+        )};                                               \
+        simd_args_sequence_free(req_args, 2);             \
+        req_args[0].data = r;                             \
+        req_args[0].dtype = simd_data_##RET;              \
+        return simd_arg_to_obj(req_args);                 \
```

Comment:
Putting the arguments in the `req_args` array is needless complexity here - may as well just declare separate variables now:
```suggestion
        simd_arg in0 = {.dtype = simd_data_##IN0};        \
        simd_arg in1 = {.dtype = simd_data_##IN1};        \
        if (!PyArg_ParseTuple(                            \
            args, "O&O&:"NPY_TOSTRING(NAME),              \
            simd_arg_converter, &in0,                     \
            simd_arg_converter, &in1                      \
        )) return NULL;                                   \
        simd_data r = {.RET = npyv_##NAME(                \
            in0.data.IN0,                         \
            in1.data.IN1                          \
        )};                                               \
        simd_arg_free(&in1);             \
        simd_arg_free(&in0);             \
        simd_arg ret = {.dtype = simd_data_##RET, .data = r };\
        return simd_arg_to_obj(&ret);
```

Reply:
(note also that it's easier to follow if you declare a new variable for `ret`, rather than reusing the one you had before)

Reply:
done